src/ZfSnapPhpDebugBar/Service/PhpDebugBarFactory.php

```diff
@@ -38,7 +38,7 @@ public function createService(Locator $serviceLocator)
         $debugbar->addCollector(new ConfigCollector($appConfig, 'ApplicationConfig'));
 
         // Db profiler
-        if ($serviceLocator->has($dbServiceName) && isset($config['db']['driver'])) {
```

Comment:
Later we check PDO driver (here)[https://github.com/bupy7/ZfSnapPhpDebugBar/blob/master/src/ZfSnapPhpDebugBar/Service/PhpDebugBarFactory.php#L70]. It line does not make sense.